scripts/cddl/generate.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+
+const spec = path.resolve(__dirname, '..', '..', 'index.bs')
+const specContent = fs.readFileSync(spec, 'utf-8')
+
+const preFormattedTexts = [...specContent.match(/<pre .*>([^<]*)<\/pre>/g)]
```

Comment:
Parsing HTML with a regex makes me very nervous :) Is there some chance to use e.g. [1] or some other spec-compliant parser for parsing instead? 

[1] https://www.npmjs.com/package/parse5